src/components/navbar/Navbar.tsx

```diff
@@ -6,7 +6,7 @@ interface NavbarProps {
 
 export default function Navbar({ children }: NavbarProps) {
   return (
-    <nav className="navbar bg-base-100">
+    <nav className="navbar flex bg-base-100 h-[75px]">
```

Comment:
was this necessary for your current task? If not, please revert this. There also seems to be an issue in this branch where I can't click on the signout button.

Reply:
Yea it is needed just for the current task, otherwise will be difficult to fix the style of the sidebar, can I keep it? I think it could be removed with the refactor of the navbar

Reply:
The issue about the signout button is related to the navbar, I can fix it but I think it should be handled in the navbar refactor

Reply:
@Dan-Y-Ko also the active state of the neutral variant is a bit dark, is this expected?

Reply:
Sure, you can keep the nav refactors for this pr.